src/Components/Promises/PromiseForm/index.js

```diff
@@ -65,6 +65,7 @@ export class PromiseForm extends React.Component {
 
   render() {
     const { parties, politicians } = this.props;
+    if (!politicians) return null;
```

Comment:
can we get rid of the wrapping `div` from the `render` function?

Reply:
yes that sounds good!